UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -755,7 +749,25 @@ <h3>Additional accessibility information</h3>
 				<section id="add-info-examples">
 					<h2>Examples</h2>
 
-					<p>TBD</p>
+					<aside class="example" title="Descriptive explanations">
+						<ul>
+							<li>Content is enhanced with ARIA roles to optimize organization and facilitate navigation.</li>
+							<li>Videos included in publications have closed captions and sign language representation.</li>
+							<li>Tactile graphics have been integrated to facilitate access to visual elements by blind people.</li>
+							<li>Music scores are provided that are visually accessible.</li>
+							<li>Content contains information encoded through colors, which may be imperceptible to those who cannot distinguish colors.</li>
```

Comment:
The 4 first lines are documenting Accessibility improvements while this one is documenting a limitation.

Reply:
I'm wondering if we could find enough negative statements to fill up a "known limitations" section.